src/solvers/QAOA.py

```diff
@@ -182,7 +182,7 @@ def run(self, mapped_problem: any, device_wrapper: any, config: Config, **kwargs
         # cycles = np.arange(1, tracker['count'])
         # optim_classical = tracker['global_energies']
 
-        ## TODO maybe save this plot
+        # TODO maybe save this plot
```

Comment:
Comments beginning with "TODO" are different from normal comments. They should remain indicated with "##" in my opinion.